src/test/java/uk/gov/hmcts/reform/sandl/snlapi/security/services/S2SAuthenticationServiceTest.java

```diff
@@ -55,6 +57,9 @@ public void tokenCreator_createToken_forRules_containsProperFields() {
         final String serviceName = (String) claims.get("service");
         assertThat(SERVICE_NAME_SNL_API).isEqualTo(serviceName);
 
+        final String userName = (String) claims.get("user");
+        assertThat(USER_NAME_SNL_API).isEqualTo(userName);
```

Comment:
@leszekgu sure the best to add this to the existing integration tests (if there were some for it), therefore added the check here.

Reply:
👍